<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>519</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1055</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabPrinicpal">
       <attribute name="title">
        <string>Principal</string>
       </attribute>
       <widget class="QGroupBox" name="groupBoxDatosRX">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>200</y>
          <width>280</width>
          <height>236</height>
         </rect>
        </property>
        <property name="title">
         <string>Datos Recibidos/Transmitidos</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextEdit" name="textEditRecibido"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButtonEnroll">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>260</y>
          <width>201</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>ENROLAR/BORRAR</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBoxConex">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>300</width>
          <height>139</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Conexi√≥n</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="labelPort">
           <property name="text">
            <string>Puerto</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="comboBoxPuertos">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRefreshPorts">
             <property name="toolTip">
              <string>Refrescar puertos</string>
             </property>
             <property name="statusTip">
              <string>Refrescar puertos</string>
             </property>
             <property name="text">
              <string>Actualizar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelEstadoConex">
           <property name="text">
            <string>Estado</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditEstado">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButtonConectar">
           <property name="text">
            <string>Conectar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBoxBienvenidos">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>0</y>
          <width>571</width>
          <height>191</height>
         </rect>
        </property>
        <property name="title">
         <string>Ingresados/as</string>
        </property>
        <widget class="QTextEdit" name="textEditBienvenido">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>551</width>
           <height>151</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:45pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
